### ISSUE TITLE
Create alternative checkbox form component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#90]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To see the checkboxes rendered well with Materialize CSS. See https://materializecss.com/checkboxes.html

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via bcf9d42b
